src/GitHub_Updater/Rest_Update.php

```diff
@@ -278,5 +326,6 @@ private function get_webhook_source() {
 				break;
 		}
 		$this->upgrader_skin->messages[] = $webhook_source;
```

Comment:
In most recent commit I changed this part. It’s not necessary to return the variable as it’s no longer used.

Reply:
ok, now I saw it